scripts/helmAcs.sh

```diff
@@ -143,6 +148,7 @@ else
       --set postgresql.enabled=false \
       --set database.external=true \
       --set repository.environment.JAVA_OPTS=" -Dopencmis.server.override=true -Dopencmis.server.value=https://$EXTERNAL_NAME -Dalfresco.restApi.basicAuthScheme=true -Dsolr.base.url=/solr -Dsolr.secureComms=none -Dindex.subsystem.name=solr6 -Dalfresco.cluster.enabled=true -Ddeployment.method=HELM_CHART -Xms2000M -Xmx2000M" \
+      --set repository.environment.CATALINA_OPTS="-agentpath:/usr/local/YourKit-JavaProfiler-2018.04/bin/linux-x86-64/libyjpagent.so=sessionname=$$HOSTNAME,dir=/tmp/Alfresco/yourkit,$YOURKIT_OPTS" \
```

Comment:
I don't see an update to a newer version of the helm chart and/or docker image in this PR so I presume the referenced files already in the current docker image?

Reply:
At this point the work is not finished on the helm chart. The issue will be postponed for a couple of sprints. The intention with this PR is to capture what we have right now

Reply:
Ah OK, apologies, I missed the "WIP" prefix on the PR